<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VisaDAQControlWin</class>
 <widget class="QMainWindow" name="VisaDAQControlWin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1790</width>
    <height>963</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AFG3000 Control</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="grpAFG3252">
    <property name="geometry">
     <rect>
      <x>1150</x>
      <y>520</y>
      <width>281</width>
      <height>166</height>
     </rect>
    </property>
    <property name="title">
     <string>AFG3252 Setting</string>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="3">
      <widget class="QDoubleSpinBox" name="boxAFGLow">
       <property name="minimum">
        <double>-5000.000000000000000</double>
       </property>
       <property name="maximum">
        <double>5000.000000000000000</double>
       </property>
       <property name="value">
        <double>0.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>High</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Frequency: </string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Waveform:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QDoubleSpinBox" name="boxAFGHigh">
       <property name="minimum">
        <double>-5000.000000000000000</double>
       </property>
       <property name="maximum">
        <double>5000.000000000000000</double>
       </property>
       <property name="value">
        <double>50.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="3" column="3">
      <widget class="QLineEdit" name="lineAFGFreq">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>1</string>
       </property>
      </widget>
     </item>
     <item row="1" column="4">
      <widget class="QLabel" name="label_12">
       <property name="text">
        <string>mV</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="4">
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>mV</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="4" column="3" colspan="2">
      <widget class="QComboBox" name="boxAFGWaveform"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_11">
       <property name="text">
        <string>Low</string>
       </property>
      </widget>
     </item>
     <item row="3" column="4">
      <widget class="QComboBox" name="boxAFGFreqUnit">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_13">
       <property name="text">
        <string>Offset</string>
       </property>
      </widget>
     </item>
     <item row="2" column="3">
      <widget class="QDoubleSpinBox" name="boxAFGLow_2">
       <property name="minimum">
        <double>-5000.000000000000000</double>
       </property>
       <property name="maximum">
        <double>5000.000000000000000</double>
       </property>
       <property name="value">
        <double>0.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="2" column="4">
      <widget class="QLabel" name="label_14">
       <property name="text">
        <string>mV</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QTabWidget" name="tabTestChoosen">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>571</width>
      <height>841</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>CITIROC VBias Test</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox_4">
      <property name="geometry">
       <rect>
        <x>9</x>
        <y>541</y>
        <width>297</width>
        <height>203</height>
       </rect>
      </property>
      <property name="title">
       <string>Bias DAC Voltage Test</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_6">
       <item row="0" column="0">
        <widget class="QPushButton" name="btnDACVPath">
         <property name="text">
          <string>Path</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1" colspan="2">
        <widget class="QLineEdit" name="lineChMea">
         <property name="enabled">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="7" column="0" colspan="3">
        <widget class="QPushButton" name="btnStopVTest">
         <property name="text">
          <string>Stop V Test</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_17">
         <property name="text">
          <string>Sample:</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QSpinBox" name="boxVSamplePoints">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="value">
          <number>40</number>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_20">
         <property name="text">
          <string>Current Channel:</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0" colspan="3">
        <widget class="QPushButton" name="btnStartVTest">
         <property name="text">
          <string>Start DAC V Test</string>
         </property>
        </widget>
       </item>
       <item row="3" column="2">
        <widget class="QLabel" name="label_18">
         <property name="text">
          <string>points</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="lblDACVFileName">
         <property name="text">
          <string>Data</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QLabel" name="lblFileSuf_3">
         <property name="text">
          <string>*.txt</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="6" column="0" colspan="3">
        <widget class="QPushButton" name="btnNextChReady">
         <property name="text">
          <string>Next Channel Ready</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QGroupBox" name="groupBox_11">
      <property name="geometry">
       <rect>
        <x>9</x>
        <y>144</y>
        <width>427</width>
        <height>391</height>
       </rect>
      </property>
      <property name="title">
       <string>Bias Setting List</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_8">
       <item row="5" column="0">
        <widget class="QPushButton" name="btnGenerateListVB">
         <property name="text">
          <string>Generate List</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_23">
         <property name="text">
          <string>CITIROC Bias:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0" colspan="2">
        <widget class="QLabel" name="label_22">
         <property name="text">
          <string>Channels:</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0" colspan="3">
        <widget class="QLineEdit" name="lineSelectChannelVB">
         <property name="text">
          <string>[0:1:31]</string>
         </property>
        </widget>
       </item>
       <item row="5" column="2">
        <widget class="QPushButton" name="btnClearListVB">
         <property name="text">
          <string>Clear List</string>
         </property>
        </widget>
       </item>
       <item row="6" column="2">
        <widget class="QGroupBox" name="groupBox_15">
         <property name="title">
          <string>CITIROC Channel List</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_18">
          <item row="0" column="0">
           <widget class="QListWidget" name="listChVB"/>
          </item>
         </layout>
        </widget>
       </item>
       <item row="4" column="0" colspan="3">
        <widget class="Line" name="line_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QGroupBox" name="groupBox_14">
         <property name="title">
          <string>CITIROC Bias List</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_17">
          <item row="0" column="0">
           <widget class="QListWidget" name="listBias"/>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="0" colspan="3">
        <widget class="QLineEdit" name="lineCITIROCBias">
         <property name="text">
          <string>100</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QGroupBox" name="groupBox_7">
      <property name="geometry">
       <rect>
        <x>9</x>
        <y>9</y>
        <width>427</width>
        <height>101</height>
       </rect>
      </property>
      <property name="title">
       <string>Device Connection</string>
      </property>
      <widget class="QPushButton" name="pushButton">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>151</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>Check Device Status</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tabCITIROCNL">
     <attribute name="title">
      <string>CITIROC AMP NL Test</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="geometry">
       <rect>
        <x>9</x>
        <y>655</y>
        <width>144</width>
        <height>89</height>
       </rect>
      </property>
      <property name="title">
       <string>CITIROC NL Test</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_5">
       <item row="0" column="0">
        <widget class="QPushButton" name="btnADCNL">
         <property name="text">
          <string>Start NL Test</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QPushButton" name="btnStopNL">
         <property name="text">
          <string>Stop after this done</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>9</x>
        <y>9</y>
        <width>427</width>
        <height>640</height>
       </rect>
      </property>
      <property name="title">
       <string>Amp &amp;&amp; Gain List</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="6" column="0" colspan="3">
        <widget class="Line" name="line">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="7" column="2">
        <widget class="QPushButton" name="btnClearListNL">
         <property name="text">
          <string>Clear List</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0" colspan="2">
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>AFG Amplitude/Offset:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0" colspan="3">
        <widget class="QLineEdit" name="lineAFGAmp">
         <property name="text">
          <string>[50:10:100]</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0" colspan="3">
        <widget class="QLineEdit" name="lineCITIROCGain">
         <property name="text">
          <string>30</string>
         </property>
        </widget>
       </item>
       <item row="9" column="2">
        <widget class="QGroupBox" name="groupBox_16">
         <property name="title">
          <string>CITIROC Channel List</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_19">
          <item row="0" column="0">
           <widget class="QListWidget" name="listChNL"/>
          </item>
         </layout>
        </widget>
       </item>
       <item row="9" column="0">
        <widget class="QGroupBox" name="groupBox_6">
         <property name="title">
          <string>CITIROC Gain List</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_9">
          <item row="0" column="0">
           <widget class="QListWidget" name="listGain"/>
          </item>
         </layout>
        </widget>
       </item>
       <item row="7" column="0">
        <widget class="QPushButton" name="btnGenerateListNL">
         <property name="text">
          <string>Generate List</string>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QComboBox" name="boxHGLG">
         <item>
          <property name="text">
           <string>High Gain</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Low Gain</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="8" column="0" colspan="3">
        <widget class="QGroupBox" name="groupBox_5">
         <property name="title">
          <string>TEK AFG List</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_7">
          <item row="0" column="0">
           <widget class="QListWidget" name="listAmp"/>
          </item>
         </layout>
        </widget>
       </item>
       <item row="2" column="0" colspan="2">
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>CITIROC Amp Gain:</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_24">
         <property name="text">
          <string>Channels:</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0" colspan="3">
        <widget class="QLineEdit" name="lineSelectChannelNL">
         <property name="text">
          <string>[0:1:31]</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_9">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>30</y>
      <width>481</width>
      <height>831</height>
     </rect>
    </property>
    <property name="title">
     <string>Device Status</string>
    </property>
    <widget class="QGroupBox" name="grpDAQctrl">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>210</y>
       <width>321</width>
       <height>311</height>
      </rect>
     </property>
     <property name="title">
      <string>FEE DAQ Control</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_23">
      <item row="1" column="0">
       <widget class="QGroupBox" name="grpDAQStart">
        <property name="title">
         <string>DAQ Setting</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_12">
         <item row="0" column="1">
          <widget class="QSpinBox" name="boxDAQEvent">
           <property name="minimum">
            <number>-1</number>
           </property>
           <property name="maximum">
            <number>999999999</number>
           </property>
           <property name="value">
            <number>-1</number>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QTimeEdit" name="timeDAQSetting">
           <property name="displayFormat">
            <string>HH:mm:ss</string>
           </property>
           <property name="time">
            <time>
             <hour>0</hour>
             <minute>0</minute>
             <second>0</second>
            </time>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QSpinBox" name="boxBufferWait">
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>999999999</number>
           </property>
           <property name="value">
            <number>200</number>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_19">
           <property name="text">
            <string>Time Setting:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label_32">
           <property name="text">
            <string>nEvents Setting:</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Loop Wait (ms)</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Loop Least Events:</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QSpinBox" name="boxLeastEvents">
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>1000</number>
           </property>
           <property name="value">
            <number>30</number>
           </property>
          </widget>
         </item>
         <item row="4" column="0" colspan="2">
          <widget class="QCheckBox" name="boxClearQueue">
           <property name="text">
            <string>Clear Q before DAQ</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QPushButton" name="pushButton_2">
        <property name="text">
         <string>Check FEE Connnection</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QGroupBox" name="groupBox_10">
        <property name="title">
         <string>Data File</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_10">
         <item row="0" column="0">
          <widget class="QPushButton" name="btnPath">
           <property name="text">
            <string>Choose Path</string>
           </property>
          </widget>
         </item>
         <item row="0" column="3">
          <widget class="QLabel" name="lblFileSuf_2">
           <property name="text">
            <string>*.root</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="0" column="1" colspan="2">
          <widget class="QLineEdit" name="lblFileName">
           <property name="text">
            <string>Data</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="2" column="0">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>1460</x>
      <y>440</y>
      <width>301</width>
      <height>121</height>
     </rect>
    </property>
    <property name="title">
     <string>Bias DAC R Test</string>
    </property>
    <layout class="QGridLayout" name="gridLayout_4">
     <item row="0" column="0">
      <widget class="QLabel" name="label_10">
       <property name="text">
        <string>Waiting:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QSpinBox" name="boxRTestTime">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="value">
        <number>1</number>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="label_15">
       <property name="text">
        <string>s</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0" colspan="3">
      <widget class="QPushButton" name="btnStartRTest">
       <property name="text">
        <string>Start DAC R Test</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0" colspan="3">
      <widget class="QPushButton" name="btnStopRTest">
       <property name="text">
        <string>Stop R Test</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1790</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
